paths/electron/packaging-apps/02-using-electron-packager.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Using electron-packager
+permalink: /electron/packaging-apps/using-electron-packager
+next-page: /electron/packaging-apps/adding-an-icon
+facilitator: false
+sidebar:
+  nav: "electron-packaging-apps"
+main-content: |
+
+  ## Using `electron-packager`
+
+  Now that we have installed `electron-packager`, we're ready to use it. We could type out the entire command every time, but it's best practice to add a build script to the `package.json` file. This means that every time we run the default build script, our specific list of commands will be run.
+
+  1. In the `package.json` file, add a build script `"build": "electron-packager . app-name --ignore=node_modules/electron-*"` in "scripts" of package.json. (Ignore all dependency apps the same way.)
+  1. Run the new build script with `npm run build`
```

Comment:
This is probably due to whatever errors I had back in the getting started course, but I get an error:

<img width="1178" alt="screen shot 2017-08-04 at 5 36 47 pm" src="https://user-images.githubusercontent.com/16547949/28987937-8809c7a4-793b-11e7-95dc-c9aea721de24.png">